### DISCUSSION TITLE
Column reordering and hide or show option

### DISCUSSION THREAD

**Discussion by superset-user** [apache/superset#25220] [Q&A / Help]
Hi everyone, I would like to know whether these features/options are available in superset 2.1.0 on dashboard:
1. Reposition (may be drag & drop, etc) columns basis user preference, e.g. when viewing a table on dashboard, a user can reposition column to start or end according to his/her preference
2. Hide or show dashboard table columns basis user preference.

I know this can be done using chart settings (but reflects for all users), but I want to know whether this can be possible for particular user (temporarily) while accessing a dashboard. Same changes shouldn't reflect for other users.

**Comment by superset-user** [comment]
For point 1, I found that new superset version 2.1.0 (not present in 1.4.2) provides an option (ALLOW COLUMNS TO BE REARRANGED) in chart settings. 

Any help on point 2 will be highly appreciated. Thanks in advance!

**Reply by vidueirof** [reply]
Have you solved this @superset-user ?

**Reply by ekane3** [reply]
Have you solved point 2 ? 🙂
